libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Connector.Teams;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Teams
+{
+    public class TeamsInfo
+    {
+        private string _conversationId;
+        private IConnectorClient _connectorClient;
+        private ITeamsConnectorClient _teamsConnectorClient;
+        private TeamsChannelData _teamsChannelData;
+
+        public TeamsInfo(string conversationId, ConnectorClient connectorClient, TeamsChannelData teamsChannelData)
```

Comment:
this might not be right - probably better to pass in teamId rather than TeamChannelData...